### PR TITLE
update sprint branch and lint fixes

### PR DESCRIPTION
**Pull request by cnardi-dev**

### CODE REVIEW

File: ArgusWeb/app/js/services/charts/chartElements.js [hunk 1]
```diff
@@ -312,7 +312,7 @@ angular.module('argus.services.charts.elements', [])
 	};
 
 	this.appendFlagsElements = function (svg_g, chartId) {
-		var flags = svg_g.append('g').attr('class', 'flags');
+		// var flags = svg_g.append('g').attr('class', 'flags');
```
Comment (xizi-xu) [inline ArgusWeb/app/js/services/charts/chartElements.js]: We need to keep the flags here; cannot comment it out.

File: ArgusWeb/app/js/services/charts/chartElements.js [hunk 2]
```diff
@@ -394,7 +394,7 @@ angular.module('argus.services.charts.elements', [])
 		chart.append('path')
 			.attr('class', 'line ' + metric.graphClassName)
 			.style('stroke', color)
-			.style('clip-path', "url('#clip_'" + chartId + "'')")
```
Comment (xizi-xu) [inline ArgusWeb/app/js/services/charts/chartElements.js]: I don't think this was correct initially. The correct version of this line should be `.style('clip-path', "url('#clip_" + chartId + "')");` before lint formatting. Same thing applies to other apperences.

File: ArgusWeb/app/js/services/charts/chartElements.js [hunk 3]
```diff
@@ -516,7 +516,7 @@ angular.module('argus.services.charts.elements', [])
 		// add the info box while hovering over
 		label.on('click', function () {
 				// click to make the label tip stay while hovering over and enlarge the annotation's circle
-				if (label.attr('clicked') !== 'Yes';) {
+				if (label.attr('clicked') !== 'Yes') {
```
Comment (xizi-xu) [inline ArgusWeb/app/js/services/charts/chartElements.js]: Thanks!

File: ArgusWeb/app/js/services/charts/chartElements.js (hunk 2 above)
Comment (cnardi-dev) [inline ArgusWeb/app/js/services/charts/chartElements.js]: Ok, will take another look.

File: ArgusWeb/app/js/services/charts/chartElements.js (hunk 1 above)
Comment (xizi-xu) [inline ArgusWeb/app/js/services/charts/chartElements.js]: Can you uncomment this before you merge the pr? @cnardi-dev

File: ArgusWeb/app/js/services/charts/chartElements.js (hunk 2 above)
Comment (cnardi-dev) [inline ArgusWeb/app/js/services/charts/chartElements.js]: updated the corrections is 2nd commit.

**Pull request merged**